src/main/webapp/app/modules/home/home.scss

```diff
@@ -30,3 +30,8 @@ ul.custom {
     width: 100%;
   }
 }
+
+a {
+  color: #005ea2;
+  font-weight: bold;
+}
```

Comment:
Move this CSS to it is own confirmation.scss file

Reply:
Done.